sdl_viewer/src/main.rs

```diff
@@ -302,10 +308,18 @@ fn main() {
                     .help("Input directory of the octree directory to serve.")
                     .index(1)
                     .required(true),
+                clap::Arg::with_name("cache_size_mb")
+                    .help("Maximum cache size in MB for octree nodes in GPU memory")
+                    .required(false)
             ]
         )
         .get_matches();
 
+    // Maximum number of MB for the octree node cache in range 1..16 GB. The default is 2 GB
+    let max_mb_nodes: usize = cmp::min(cmp::max(1000, matches.value_of("cache_size_mb").unwrap_or("2000").parse().unwrap()), 16000);
```

Comment:
this now is hard to read. suggestion:

~~~
let max_mb_nodes = {
     let value = matches.value_of("cache_size_mb").unwrap_or("2000").parse().unwrap();
     cmp::min(cmp::max(value, 1000), 16000))
};
~~~